### PR TITLE
Change to point to J8 Quickstart

### PR DESCRIPTION
**Pull request by lesv**
Please merge after approval.

### CODE REVIEW

File: APPENGINE.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Use Google Cloud Java Client with Google App Engine
+Use Google Cloud Java Client with Google App Engine Java 8
```
Comment (garrettjonesgoogle) [inline APPENGINE.md]: I'd prefer not to have the title mention Java 8; that implies that Java 7 instructions are somewhere else. We should probably have the content below state that Java 7 is not supported.

**Comment by lesv** [comment]
PTAL

File: APPENGINE.md (hunk 1 above)
Comment (garrettjonesgoogle) [reply APPENGINE.md]: Can we remove the "Java 8" from the title here?

Comment (lesv) [reply APPENGINE.md]: I was trying to re-enforce this is the way.  I'm not a fan of subtle messages when we are getting issues about this.

Comment (garrettjonesgoogle) [reply APPENGINE.md]: Actually I just noticed that the title and the 2nd note below conflict... the title says these are instructions for Java 8, and the note says that the instructions aren't required for Java 8.

**Comment by garrettjonesgoogle** [comment]
@lesv do you want to address my most recent feedback so that this PR can be completed?

**Comment by lesv** [comment]
@garrettjonesgoogle PTAL

**Comment by garrettjonesgoogle** [comment]
I feel like this file can be entirely deleted now that it is so stripped down. You would also need to update the main README which currently points to this file.

**Pull request closed without merging**